frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -1,21 +1,37 @@
 import * as React from 'react';
 import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import * as _ from 'lodash';
 import { sortable } from '@patternfly/react-table';
-import { getNodeMachineNameAndNamespace, getName, getUID } from '@console/shared';
-import { MachineModel, NodeModel } from '@console/internal/models';
+import { getName, getUID } from '@console/shared';
+import { NodeModel } from '@console/internal/models';
 import { NodeKind, referenceForModel } from '@console/internal/module/k8s';
 import { Table, TableRow, TableData, ListPage } from '@console/internal/components/factory';
-import { Kebab, ResourceKebab, ResourceLink } from '@console/internal/components/utils';
+import {
+  Kebab,
+  ResourceKebab,
+  ResourceLink,
+  humanizeBinaryBytes,
+  formatCores,
+} from '@console/internal/components/utils';
+import { NodeMetrics, setNodeMetrics } from '@console/internal/actions/ui';
+import { PROMETHEUS_BASE_PATH } from '@console/internal/components/graphs';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { fromNow } from '@console/internal/components/utils/datetime';
 import { nodeStatus } from '../../status/node';
 import NodeRoles from './NodeRoles';
 import { menuActions } from './menu-actions';
 import NodeStatus from './NodeStatus';
 
 const tableColumnClasses = [
-  classNames('col-md-4', 'col-sm-4', 'col-xs-8'),
-  classNames('col-md-3', 'col-sm-4', 'col-xs-4'),
-  classNames('col-md-2', 'col-sm-4', 'hidden-xs'),
-  classNames('col-md-3', 'hidden-sm', 'hidden-xs'),
+  classNames('col-md-3', 'col-sm-4', 'col-xs-8', 'col-lg-2'),
+  classNames('col-md-1', 'col-sm-4', 'col-xs-4', 'col-lg-1'),
+  classNames('col-md-1', 'col-sm-4', 'hidden-xs', 'col-lg-1'),
+  classNames('col-md-1', 'hidden-sm', 'hidden-xs', 'col-lg-1'),
+  classNames('hidden-md', 'hidden-sm', 'hidden-xs', 'col-lg-2'),
+  classNames('hidden-md', 'hidden-sm', 'hidden-xs', 'col-lg-1'),
+  classNames('hidden-md', 'hidden-sm', 'hidden-xs', 'col-lg-2'),
+  classNames('col-md-1', 'hidden-sm', 'hidden-xs', 'col-lg-2'),
```

Comment:
We have too many columns to use Bootstrap grid. Remove the grid classes as in #3669

Reply:
ack. made similar changes to 3669. I will send a follow-up PR using PF sizing css, if it's okay with you.